src/component/mxgraph/shape/edges.ts

```diff
@@ -33,12 +33,12 @@ export class BpmnConnector extends mxgraph.mxConnector {
     c.setShadow(false);
     c.setDashed(false, false);
 
-    if (sourceMarker != null) {
+    if (sourceMarker != undefined) {
```

Comment:
**question:** while not using `if (sourceMarker)` (same remark for other check using the same pattern)
Checking _falsability_ covers both null and undefined for objects